package.json

```diff
@@ -131,6 +131,10 @@
           "type": "boolean",
           "markdownDescription": "When this option is enabled, for each method that has implicit arguments they are displayed as additional decorations."
         },
+        "metals.showImplicitConversionsAndClasses": {
+          "type": "boolean",
+          "markdownDescription": "When this option is enabled, for each method that has implicit arguments they are displayed as additional decorations."
```

Comment:
I'm assuming you don't want the copied description from up above? 😆

Reply:
Argh, I haven't actually saved the file before pushing :sweat_smile:

Reply:
Fixed!